com.eminfedar.vaktisalah.json

```diff
@@ -0,0 +1,42 @@
+{
+    "id": "com.eminfedar.vaktisalah",
+    "branch": "master",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "vaktisalah",
+    "finish-args": [
+        "--share=network",
+        "--socket=pulseaudio",
+        "--share=ipc",
+        "--socket=x11",
+        "--filesystem=home"
```

Comment:
What does it need home access for?

Reply:
It stores configuration file at `~/.config/vaktisalah/settings.json` which stores the window's position, selected country etc.

Reply:
use `--persist` then

Reply:
I have removed filesystem permission completely and it is still working as expected. So I can remove it I think. Because it seems no permission required to create file at AppConfig path.